backup-stores/s3/src/main/java/io/camunda/zeebe/backup/s3/Metadata.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.backup.s3;
+
+import io.camunda.zeebe.backup.api.Backup;
+import java.util.Set;
+
+/**
+ * Holds all metadata information about a backup, for example the {@link
+ * io.camunda.zeebe.backup.api.BackupIdentifier}, the {@link
+ * io.camunda.zeebe.backup.api.BackupDescriptor} and file names of segments and snapshot.
+ *
+ * <p>Used for JSON serialization.
+ */
+record Metadata(
```

Comment:
:x: Add @JsonIgnoreProperties(ignoreUnknown = true) for backward compatibility when adding new fields.

Reply:
I think that's forwards compatability, right? Old brokers can read metadata from newer brokers where new fields are added. Anyway, added the annotation :+1: